app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -20,7 +20,9 @@ def initialize(cfg = {})
       'hawkular_datasource_remove.ok',
       'hawkular_deployment.ok',
       'hawkular_deployment_remove.ok',
-      'hawkular_event' # # general purpose detail level event
+      'hawkular_event', # general purpose detail level event
+      # filtered (not shown in timeline)
```

Comment:
@bronaghs Good question but no, we need both.  The issue was that the 'hawkular_event' event_type was being used both by the alerts mechanism (for internal use) *and* for legitimate events to be shown on the timeline.  With this change the 'hawkular_alert' event_type will now be used exclusively for alerting, and not shown on the timeline, leaving the valid 'hawkular_event' events to be shown.